<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2018-02-17T10:53:24Z" name="Suite" finished-at="2018-02-17T10:53:47Z" duration-ms="23203">
    <groups>
    </groups>
    <test started-at="2018-02-17T10:53:24Z" name="Validating Customer Logins" finished-at="2018-02-17T10:53:39Z" duration-ms="15392">
      <class name="com.gowri.End2End.Tc1CustomersLogin">
        <test-method is-config="true" signature="initialize()[pri:0, instance:com.gowri.End2End.Tc1CustomersLogin@57abad67]" started-at="2018-02-17T10:53:24Z" name="initialize" finished-at="2018-02-17T10:53:27Z" duration-ms="3294" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="customerLogin()[pri:0, instance:com.gowri.End2End.Tc1CustomersLogin@57abad67]" started-at="2018-02-17T10:53:27Z" name="customerLogin" finished-at="2018-02-17T10:53:39Z" duration-ms="11370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- customerLogin -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.gowri.End2End.Tc1CustomersLogin@57abad67]" started-at="2018-02-17T10:53:39Z" name="teardown" finished-at="2018-02-17T10:53:39Z" duration-ms="709" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.gowri.End2End.Tc1CustomersLogin -->
    </test> <!-- Validating Customer Logins -->
    <test started-at="2018-02-17T10:53:40Z" name="Validating Manager Logins" finished-at="2018-02-17T10:53:47Z" duration-ms="7777">
      <class name="com.gowri.End2End.Tc2ManagerLogin">
        <test-method is-config="true" signature="initialize()[pri:0, instance:com.gowri.End2End.Tc2ManagerLogin@4b41e4dd]" started-at="2018-02-17T10:53:40Z" name="initialize" finished-at="2018-02-17T10:53:41Z" duration-ms="1981" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="managerLogin()[pri:0, instance:com.gowri.End2End.Tc2ManagerLogin@4b41e4dd]" started-at="2018-02-17T10:53:41Z" name="managerLogin" finished-at="2018-02-17T10:53:46Z" duration-ms="4690" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [XYZ Bank] but found [XYZ Bank1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [XYZ Bank] but found [XYZ Bank1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.gowri.End2End.Tc2ManagerLogin.managerLogin(Tc2ManagerLogin.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:373)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:334)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:407)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- managerLogin -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:com.gowri.End2End.Tc2ManagerLogin@4b41e4dd]" started-at="2018-02-17T10:53:47Z" name="teardown" finished-at="2018-02-17T10:53:47Z" duration-ms="770" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.gowri.End2End.Tc2ManagerLogin -->
    </test> <!-- Validating Manager Logins -->
  </suite> <!-- Suite -->
</testng-results>
